### PR TITLE
apps sc: synced timezone in all grafana dashboards

### LINKED ISSUE #1297: Standardize Grafana dashboard timezones
**Is your feature request related to a problem? Please describe.**

The default timezone in each of our Grafana dashboards are quite inconsistent, and they can't be changed through the Grafana GUI since they were created by another source. So when you update timezone in a dashboard through the top right time picker it doesn't persist and gets reset every time you refresh the page. Maybe the most frustrating part is that we have different timezones in different dashboards.

**Describe the solution you'd like**

Change the default timezone in all of our dashboards to default organization timezone, which is just an empty string in the JSON model `""`

Any Grafana admin can change the default organization timezone, this means that the timezone will be the same for every user that is logged in.

**Describe alternatives you've considered**

An alternative was to change the timezones to "browser" timezone, but this can cause confusion if people are in different timezones.

**Additional context**

Screenshot from 2022-11-15 11-13-48

**Definition of done:**

All of our Grafana dashboards are using the same timezone.

### PR DESCRIPTION
**Pull request by davidumea**
**What this PR does / why we need it**:

**Which issue this PR fixes** : fixes #1297

**Checklist:**

- [X] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [X] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [X] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [X] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

### CODE REVIEW

**Review by raviranjanelastisys — APPROVED** [review]

File: helmfile/upstream/kube-prometheus-stack/values.yaml [hunk 1]
```diff
@@ -640,7 +640,7 @@ grafana:
   ## Timezone for the default dashboards
   ## Other options are: browser or a specific timezone, i.e. Europe/Luxembourg
   ##
-  defaultDashboardsTimezone: utc
+  defaultDashboardsTimezone: ""
```
Comment (crssnd) [inline helmfile/upstream/kube-prometheus-stack/values.yaml:643]: not necessary as I see you already modified the `helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl`

Comment (davidumea) [reply helmfile/upstream/kube-prometheus-stack/values.yaml:643]: Why not have it the same everywhere?

**Review by crssnd — APPROVED** [review]
looks good

File: helmfile/upstream/kube-prometheus-stack/values.yaml (hunk 1 above)
Comment (crssnd) [reply helmfile/upstream/kube-prometheus-stack/values.yaml:643]: if we update the upstream chart this file will be overwritten
we use kube-prometheus-stack-sc.yaml.gotmpl to change any of the upstream chart defaults

**Pull request merged**